bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -60,12 +60,12 @@ final class BagsManager(executor: ScheduledThreadPoolExecutor) {
    * consideration for a BAGS lookup.
    */
   def watch(prog: ISPProgram): Unit = {
-    //    synchronized {
-    //      state += prog.getProgramID
-    //      prog.addStructureChangeListener(StructurePropertyChangeListener)
-    //      prog.addCompositeChangeListener(CompositePropertyChangeListener)
-    //    }
-    //    prog.getAllObservations.asScala.foreach(enqueue(_, 0L, initialEnqueue = true))
+        synchronized {
```

Comment:
Yes, but it is as of yet unmerged, but the changes are the exact same between the two PRs (and I needed BAGS enabled for testing for this), so I suspect there will not be conflicts?